src/header/css/header-brand.pcss

```diff
@@ -70,7 +68,7 @@
 /* ========================================================================== */
 
 .esri-header-brand-text {
-	font-size: 12px;
+	font-size: 18px;
```

Comment:
This looks like it was changed for UC 2020, but it's making all the apps titles really small. Thought this was safe, but something to look at.